### PR TITLE
Update coka recipes and add new recipes

### PR DESCRIPTION
**Pull request by CedrickOka**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by Nyholm** [comment]
thank you. Could you move the parameters to the manifest file?

**Comment by CedrickOka** [comment]
Hi,

@Nyholm Can you help me by looking at my pull request?

File: coka/file-bundle/3.0/config/packages/oka_file.yaml [hunk 1]
```diff
@@ -4,4 +4,3 @@ oka_file:
         root_path: '%env(OKAFILE_STORAGE_ROOT_PATH)%'
         webserver:
             host: '%env(OKAFILE_STORAGE_WEB_SERVER_HOST)%'
-
```
Comment (Nyholm) [inline coka/file-bundle/3.0/config/packages/oka_file.yaml]: Why were these lines changed?

File: coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+oka_rest_request_validator:
+    exception:
+        enabled: true
```
Comment (Nyholm) [inline coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml]: Is this the same as the default value?

File: coka/rate-limit-bundle/1.0/config/packages/oka_rate_limit.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+oka_rate_limit:
+    configs:
+        - {path: ''}
```
Comment (Nyholm) [inline coka/rate-limit-bundle/1.0/config/packages/oka_rate_limit.yaml]: I like this config, because me as user are probably going to change that. 

:+1:

File: coka/file-bundle/3.0/config/packages/oka_file.yaml (hunk 1 above)
Comment (CedrickOka) [reply coka/file-bundle/3.0/config/packages/oka_file.yaml]: Hi,
These lines have changed to allow you to use environment variables to configure where your files are stored and little like the doctrine URL database

File: coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml (hunk 2 above)
Comment (CedrickOka) [reply coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml]: Hi,
yes it is the same thing as the default variable but I put it more in mind to indicate what needs to be changed according to these needs

File: coka/rate-limit-bundle/1.0/config/packages/oka_rate_limit.yaml (hunk 3 above)
Comment (CedrickOka) [reply coka/rate-limit-bundle/1.0/config/packages/oka_rate_limit.yaml]: Hi,
Cool, thanks

File: coka/file-bundle/3.0/config/packages/oka_file.yaml (hunk 1 above)
Comment (Nyholm) [reply coka/file-bundle/3.0/config/packages/oka_file.yaml]: If I look at the diff. I see that you just removed an empty line in this file. 

Generally you should not modify other files or fix CS. I just asked if there is any particular reason for this change or if it just was “unrelated CS”.

File: coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml (hunk 2 above)
Comment (Nyholm) [reply coka/rest-request-validator-bundle/1.0/config/packages/oka_rest_request_validator.yaml]: Hm... okey.

File: coka/file-bundle/3.0/config/packages/oka_file.yaml (hunk 1 above)
Comment (CedrickOka) [reply coka/file-bundle/3.0/config/packages/oka_file.yaml]: Well understood thank you
